test/Demangle/Inputs/manglings.txt

```diff
@@ -350,12 +350,13 @@ $S1t1PP10AssocType2_AA1QTN ---> default associated conformance accessor for t.P.
 $sSD5IndexVy__GD ---> $sSD5IndexVy__GD
 $s4test3StrCACycfC ---> {T:$s4test3StrCACycfc} test.Str.__allocating_init() -> test.Str
 $s18keypaths_inlinable13KeypathStructV8computedSSvpACTKq  ---> key path getter for keypaths_inlinable.KeypathStruct.computed : Swift.String : keypaths_inlinable.KeypathStruct, serialized
-$s18resilient_protocol24ResilientDerivedProtocolPxAA0c4BaseE0Tn --> associated conformance descriptor for resilient_protocol.ResilientDerivedProtocol.A: resilient_protocol.ResilientBaseProtocol
-$s3red4testyAA3ResOyxSayq_GAEs5ErrorAAq_sAFHD1__HCg_GADyxq_GsAFR_r0_lF --> red.test<A, B where B: Swift.Error>(red.Res<A, B>) -> red.Res<A, [B]>
+$s18resilient_protocol24ResilientDerivedProtocolPxAA0c4BaseE0Tn ---> associated conformance descriptor for resilient_protocol.ResilientDerivedProtocol.A: resilient_protocol.ResilientBaseProtocol
```

Comment:
@DougGregor -- Looks like you added this back in 76794334fab7.  But it never actually ran because it had `-->` rather than `--->`, and seems to be broken now.  Should I just put it back for now?  Any suggestions how to fix it?

Reply:
I just put it back for now.